frontend/public/components/cluster-service-class.tsx

```diff
@@ -1,16 +1,17 @@
 /* eslint-disable no-undef */
 
 import * as React from 'react';
-import { Link } from 'react-router-dom';
+import * as _ from 'lodash-es';
 
+import { Link } from 'react-router-dom';
```

Comment:
Nit:  this should go on line 5?

Reply:
Yes, we group console imports separately from other dependencies. This should be moved back where it was.